services/certificate-api/src/routes/certificate_controller.js

```diff
@@ -247,6 +248,14 @@ async function renderDataToTemplate(templateFileURL, data) {
 
 async function createPDF(certificate) {
     try {
+        if(certificate.indexOf("svg") !== -1) {
+            const doc = new PDFDocument({
+                layout: "portrait",
```

Comment:
is the layout mandatory? what if it is landscape?

Reply:
it is not mandatory but by default the value is portrait and A4